### ISSUE TITLE
Create UI Design

### ISSUE THREAD

**Issue by phildree** [amosproj/amos2023ss01-apache-pulsar-ui#48]
## User story
1. As a frontend dev
2. I need coherent styling guidelines
3. So that I know how to develop my components

## Acceptance criteria
Create style guidelines that all frontend dev agree on

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Shahraz98** [comment]
https://www.figma.com/file/DKnW8kRKoCGC6071oY4PAy/AMOS-Apache-Pulsar?type=design&node-id=0%3A1&t=oqv1nurDQlwIgnsQ-1
